From 11921ac7e6977d9cb153e88cc4eea8e618006a07 Mon Sep 17 00:00:00 2001
From: Vivek Goyal <vgoyal@redhat.com>
Date: Tue, 19 Aug 2014 22:03:00 -0400
Subject: [crypto] pkcs7: fix sparse non static symbol warning

Message-id: <1408485785-777-29-git-send-email-vgoyal@redhat.com>
Patchwork-id: 88383
O-Subject: [RHEL7.1 PATCH 28/33] PKCS#7: fix sparse non static symbol warning
Bugzilla: 1129779
RH-Acked-by: David Howells <dhowells@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

BZ:1129779

commit 63d2551ea7e2c34b533f2b6b0646cc825d9ee509
Author: Wei Yongjun <yongjun_wei@trendmicro.com.cn>
Date: Mon, 28 Jul 2014 21:17:12 +0800

Fixes the following sparse warnings:

crypto/asymmetric_keys/pkcs7_key_type.c:73:17: warning:
 symbol 'key_type_pkcs7' was not declared. Should it be static?

Signed-off-by: Wei Yongjun <yongjun_wei@trendmicro.com.cn>
Signed-off-by: David Howells <dhowells@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/crypto/asymmetric_keys/pkcs7_key_type.c b/crypto/asymmetric_keys/pkcs7_key_type.c
index 24a547f..310d396 100644
--- a/crypto/asymmetric_keys/pkcs7_key_type.c
+++ b/crypto/asymmetric_keys/pkcs7_key_type.c
@@ -70,7 +70,7 @@ error:
  * user defined keys take an arbitrary string as the description and an
  * arbitrary blob of data as the payload
  */
-struct key_type key_type_pkcs7 = {
+static struct key_type key_type_pkcs7 = {
  .name   = "pkcs7_test",
  .def_lookup_type = KEYRING_SEARCH_LOOKUP_DIRECT,
  .instantiate  = pkcs7_instantiate,
-- 
1.7.1